### PR TITLE
New germplasm accession importer

### PR DESCRIPTION
**Pull request by Jiu9Shen**
ReadtheDocs has been updated in module, but it's not updated on [VCF Filter: Documentation](https://vcf-filter.readthedocs.io/en/latest/) yet since its not merged to master.
ReadtheDocs can be generated locally by following [KP guide](https://knowpulse.readthedocs.io/en/latest/developer_docs/rtd.html).
Please follow ReadtheDocs, and also run a job after installation to make sure ontology files can be imported.

One example test file for accession importer is provided in module in `tripal_germplam_importer/tests/test_files/`.

How to test the module:
1. need to re-install this module
    1.a disable this module and then uninstall it
    1.b update module and change to branch 12-new_class_GermplasmAccessionImporter
    1.c enable module(run a job if you are on dev), and go to Admin->People->Permissions to give permissions to "Importer: Use the Germplasm Cross Importer" and "Importer: Use the Germplasm Accession Importer"
2. now both loaders should appear in Admin->Data Loaders, it can be tested with an example file included in the moduel in tests/test_files
3. ReadtheDocs is updated, but it required to run `make html` under directory `docs/` on terminal then open docs/_build_html/index.html

### CODE REVIEW

**Comment by Jiu9Shen** [comment]
The module can load accession files now. 
Tested on tripal, local KP and KP2/fresh alreay.

**Review by laceysanderson — CHANGES_REQUESTED** [review]
Ran the importer with your provided test file and recieved 2X the germplasm...
Screen Shot 2019-10-22 at 10 57 37 AM

```
$ drush trp-run-jobs --username=tripaladmin --root=/Users/laceysanderson/Sites/tripal-TEST

2019-10-22 11:51:25
Tripal Job Launcher
Running as user 'tripaladmin'
-------------------
2019-10-22 11:51:25: There are 1 jobs queued.
2019-10-22 11:51:25: Job ID 12.
2019-10-22 11:51:25: Calling: tripal_run_importer(9)

Running 'Germplasm Accession Importer' importer
NOTE: Loading of file is performed using a database transaction. 
If it fails or is terminated prematurely then all insertions and 
updates are rolled back and will not be found in the database

Upload file name: germplasm_accession_test1.tsv 
WARNING: The db, KnowPulse, has no match in chado:db.
WARNING: The db, ICARDA, has no match in chado:db.es.

Done.

Remapping Chado Controlled vocabularies to Tripal Terms...
Done.
Kirstins-Air:tripal_germplasm_importer laceysanderson$ drush trp-run-jobs --username=tripaladmin --root=/Users/laceysanderson/Sites/tripal-TEST

2019-10-22 11:52:03
Tripal Job Launcher
Running as user 'tripaladmin'
-------------------
2019-10-22 11:52:03: There are 1 jobs queued.
2019-10-22 11:52:03: Job ID 13.
2019-10-22 11:52:03: Calling: chado_publish_records(Array)
INFO (PUBLISH_RECORDS): There are 4 records to publish.
There are 4 records to publish.
INFO (PUBLISH_RECORDS): Successfully published 4 Germplasm Accession record(s).
Successfully published 4 Germplasm Accession record(s).
```
When I checked the published germplasm, they were perfect duplicates with both having the properties loaded in.
Screen Shot 2019-10-22 at 11 01 02 AM

File: docs/features/germplasm_accession_importer.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.. image:: accession.1.file_format.png
+
+.. note::
+
+  For column 2 (External Database), the name in file must exist in your database in chado table:db already.
+
+  For column 12 (Pedigree), pedigree information is recommended to save in format of ``maternal-parent-name/paternal-parent-name``.
+
+  For column 13 (Synonyms), multiple synonyms are allowed but must be separated by pounctuation marks. Like, ``syn1;syn2;syn3``.
```
Comment (laceysanderson) [inline docs/features/germplasm_accession_importer.rst]:
```suggestion
  For column 13 (Synonyms), multiple synonyms are allowed but must be separated by semi-colons (e.g.  ``syn1;syn2;syn3``).
```

**Review by laceysanderson — APPROVED** [review]
Functionality works now 👍 Documentation is sufficient with one small suggested change. Thanks for the new importer @Jiu9Shen!

**Pull request merged**